cmd/tsbs_load_mongo/creator.go

```diff
@@ -1,29 +1,46 @@
 package main
 
 import (
+	"context"
 	"fmt"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
 	"log"
 	"strings"
-
-	"github.com/globalsign/mgo"
-	"github.com/globalsign/mgo/bson"
 )
 
 type dbCreator struct {
-	session *mgo.Session
+	client *mongo.Client
+	ctx    context.Context
+	cancel context.CancelFunc
 }
 
 func (d *dbCreator) Init() {
+	//log.Println("tsbs_load_mongo/creator/Init")
 	var err error
-	d.session, err = mgo.DialWithTimeout(daemonURL, writeTimeout)
+	d.ctx, d.cancel = context.WithTimeout(context.Background(), writeTimeout)
+	//defer d.cancel()
+	log.Println("TRYING TO CONNECT")
+	d.client, err = mongo.Connect(d.ctx, options.Client().ApplyURI(daemonURL))
 	if err != nil {
+		log.Println("DID NOT MANAGE TO CONNECT")
 		log.Fatal(err)
+	} else {
+		err = d.client.Ping(d.ctx, readpref.Primary())
+		if err != nil {
+			log.Println("DID NOT MANAGE TO CONNECT")
+			log.Fatal(err)
+		} else {
+			log.Println("MANAGED TO CONNECT")
```

Comment:
Can you please clean up the debugging messages? I also see it uses ping to confirm the server it responding, which I think is not needed for the connection.
```suggestion
	d.ctx, d.cancel = context.WithTimeout(context.Background(), writeTimeout)
	d.client, err = mongo.Connect(d.ctx, options.Client().ApplyURI(daemonURL))
	if err != nil {
		log.Println("Can't establish connection with", daemonURL)
		log.Fatal(err)
```